### PR TITLE
MCKIN-12203 AF#91-MRQ-Ensure custom controls are keyboard accessible

### PR DESCRIPTION
**Pull request by nasirhjafri**
MCKIN-12203 AF#91-MRQ-Ensure custom controls are keyboard accessible

Description

[Issue]
The feedback icons aren't keyboard accessible since div elements do not natively receive keyboard focus.

[User Impact]
When custom elements do not provide keyboard access, keyboard only users and users of assistive technology may not be able to use or interact with the custom control. 

[Code Reference]
<div class="choice-result fa icon-2x checkmark-incorrect icon-exclamation fa-exclamation" id="result_e3e109293d1944ce80c1-1" aria-label="Incorrect" data-label_correct="Correct" data-label_incorrect="Incorrect"></div>

Environment

[Recommended Solution]
Use the <button> element to ensure keyboard focus by default.

### CODE REVIEW

**Comment by nasirhjafri** [comment]
@xitij2000 Please review.

**Comment by nasirhjafri** [comment]
@xitij2000 Reminder: to review this one, thanks.

**Comment by lgp171188** [comment]
@nasirhjafri, I am reviewing this today and hope to have a review pass done before my EOD today.

**Comment by lgp171188** [comment]
@nasirhjafri, I tested this on the Ironwood devstack by adding a problem-builder XBlock MRQ to a course and then tabbing through the page elements with the keyboard and verified that the buttons with the `choice-result` class are accessible via the keyboard and highlighted when focussed.

PB_MRQ_keyboard_accessibility

However I see a couple of issues, which are highlighted in the screenshot. The dotted border for the highlight extends slightly beyond the edges of the highlight background color and that looks odd. Also with the dotted border, the button appears to overlap a bit with the checkbox which makes it look ugly. Perhaps the margin/padding need to be updated? Increasing the margin from the browser developer tools worked as a temporary fix for me.

File: setup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 # Constants #########################################################
 
-VERSION = '3.3.12'
+VERSION = '3.3.13'
```
Comment (lgp171188) [inline setup.py]: @nasirhjafri,

```suggestion
VERSION = '3.3.14'
```

**Comment by lgp171188** [comment]
@nasirhjafri,:+1:. This is good to merge after addressing the version bump review comment
 
- [x] I tested this on the Ironwood devstack and verified that the feedback controls are keyboard accessible. I also verified that the issues raised in my previous comments are now fixed.
- [x] I read through the code
- [x] I checked for accessibility issues
- [x] ~~Includes documentation~~ NA

**Comment by nasirhjafri** [comment]
@lgp171188 Thanks for reviewing.

**Pull request merged**